database/seed.js

```diff
@@ -8,27 +8,45 @@ const pool = new Pool({
   password: 'connorhoman',
 });
 
+const names = ['Lake Almanor Lake Front Retreat',
+  'Vintage Concept Flat in Chic Neighbourhood',
+  'Relax on the Deck of a Bright Eixample Apartment',
+  'Stroll to Sagrada Familia from a Sunlit Apartment',
+  'Brownstone Studio',
+  'Bedford Loft, Williamsburg Prime',
+  'Studio Apartment Minutes Away from Downtown',
+  'Beachfront Villa Near Boardwalk',
+  'Cute Quirky Garden apt, NYC adjacent',
+  'CLOSEST GRAND CANYON B&B/South Rim',
+  'The Cabin',
+  'Take in a Dip in the Heated Pool at a Luxury SoCo Retreat'];
+const ownerTypes = ['Verified', 'none', 'Plus'];
+const cities = ['Barcelona', 'New York', 'Tokyo',
+  'Austin', 'Westwood', 'San Francisco', 'Oakland',
+  'San Diego', 'Honolulu', 'Punta Cana, Dominican Replubic'];
+
 for (let i = 0; i < 100; i += 1) {
-  const name = faker.lorem.word();
-  const city = faker.address.city();
+  const name = names[Math.floor(Math.random() * names.length)];
+  const city = cities[Math.floor(Math.random() * cities.length)];
   const avatar = faker.image.avatar();
   const owner = faker.name.firstName();
-  const ownerType = faker.lorem.word();
-  const guests = Math.floor(Math.random() * 10);
-  const beds = Math.floor(Math.random() * 5);
-  const rooms = Math.floor(Math.random() * 4);
-  const baths = Math.floor(Math.random() * 3);
+  const ownertype = ownerTypes[Math.floor(Math.random() * ownerTypes.length)];
+  const guests = Math.floor(Math.random() * 8) + 1;
+  const beds = Math.floor(Math.random() * 6) + 1;
+  const rooms = Math.floor(Math.random() * 4) + 1;
+  const baths = Math.floor(Math.random() * 3) + 1;
+  const responserate = Math.floor(Math.random() * 100);
   const description = faker.lorem.paragraphs();
   const amenities = faker.lorem.sentence();
-  const rules = faker.lorem.paragraph();
+  const rules = faker.lorem.sentence();
   const cancellation = faker.lorem.sentence();
   const related1 = faker.lorem.sentence();
   const related2 = faker.lorem.sentence();
   const related3 = faker.lorem.sentence();
-  const location = faker.address.latitude();
-  const nearbyCities = faker.lorem.word();
-  const query = `INSERT INTO listing_info (name, city, avatar, owner, ownerType, guests, beds, rooms, baths, description, amenities, rules, cancellation, related1, related2, related3, location, nearbyCities) VALUES ('${name}', '${city}', '${avatar}', '${owner}', '${ownerType}', ${guests}, ${beds}, ${rooms}, ${baths}, '${description}', '${amenities}', '${rules}', '${cancellation}', '${related1}', '${related2}', '${related3}', '${location}', '${nearbyCities}')`;
-  console.log(query);
+  const location = `${faker.address.latitude()},${faker.address.longitude()}`;
+  const nearbycities = `${cities[Math.floor(Math.random() * cities.length)]},${cities[Math.floor(Math.random() * cities.length)]},${cities[Math.floor(Math.random() * cities.length)]},${cities[Math.floor(Math.random() * cities.length)]},${cities[Math.floor(Math.random() * cities.length)]}`;
```

Comment:
it might not be a bad idea to have a `randomElement(array)` helper function or even `getRandomCity()` here.  It's just a minor suggestion as this is just a fake data gen and not going to be in action later!

Reply:
Was thinking this same thing actually! Will do.